index.css

```diff
@@ -0,0 +1,526 @@
+  height: 147px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+}
+
+.botones4 {
+  width: 49px;
+  height: 40px;
+  position: relative;
+  left: 90px;
+  padding: 10px 19px;
+  margin-left: 20px;
+}
+
+.botones4 ul {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  font-size: 0;
+  align-items: center;
+}
+
+.botones4 button {
+  background-color: rgba(255, 255, 255, 0.24);
+  color: #091e42;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+}
+
+.proyect button {
+  width: 100px;
+  height: 40px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  background: #ff6b00;
+  color: #fff;
+  margin-left: 39px;
+}
+
+.bottom-left {
+  position: absolute;
+  bottom: 8px;
+  left: 16px;
+}
+
+.dashbord {
+  position: relative;
+  bottom: 330px;
+  left: 40px;
+  width: 295px;
+  height: 81px;
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+}
+
+.texto3 {
+  position: relative;
+  width: 295px;
+  height: 87px;
+  bottom: 340px;
+  left: 40px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.botones3 {
+  position: relative;
+  bottom: 340px;
+  margin-right: 90px;
+}
+
+.botones3 button {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  background-color: rgba(255, 255, 255, 0.24);
+  color: #fff;
+}
+
+.botones3 ul {
+  display: flex;
+  font-size: 0;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.brian button {
+  position: relative;
+  bottom: 323px;
+  left: 29px;
+  width: 327px;
+  height: 60px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  background-color: #ff6b00;
+  font-size: 17px;
+  color: #fff;
+}
+
+.container {
+  position: relative;
+}
+
+.jo {
+  position: relative;
+  bottom: 250px;
+}
+
+.ran {
+  position: relative;
+  bottom: 490px;
+}
+
+.ad {
+  position: relative;
+  bottom: 740px;
+}
+
+.ria {
+  position: relative;
+  bottom: 990px;
+}
+
+.ald {
+  position: relative;
+  bottom: 1240px;
+  height: 40px;
+}
+
+.r {
+  margin-left: 8px;
+}
+
+.h {
+  margin-left: 5px;
+}
+
+.b {
+  margin-left: 5px;
+}
+
+.profesional {
+  padding-left: 40px;
+  padding-top: 18px;
+}
+
+.z {
+  position: relative;
+  bottom: 790px;
+  width: 380px;
+  height: 2000px;
+  padding-bottom: 50px;
+  display: flex;
+  flex-direction: column;
+  left: -8px;
+  background-color: #1c1a19;
+}
+
+.abouted {
+  width: 380px;
+  height: 1000px;
+
+  /* add image as background but dosnt show in code viewer side */
+
+  /* background: url(../pictures/Frame1.png) right top no-repeat; */
```

Comment:
- [ ] Please do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history. Kindly remove the comments. :+1: